README.md

```diff
@@ -58,29 +50,29 @@ you can also run the provided Docker Compose setup.
 * Backend development: Run local production (after building, it starts very fast)
   and access the organizations directly on http://localhost:3000/organizations/.
 
-  ```
-  $ docker compose -f dev.yml --profile static up
-  ```
+  
+$ docker compose -f dev.yml --profile static up
+
```

Comment:
I think this becomes less readable when you remove the code blocks, as you've done here and in similar cases elsewhere in the file.

I'm not sure if you're looking at this in a markdown previewer when you edit it, but the change you've made here means that this line of text goes from being rendered with a monospace font (code-like) to being rendered as regular text, which I find less readable.

Reply:
good point! Thanks for taking the time to explain